mis_builder/models/accounting_none.py

```diff
@@ -89,6 +89,7 @@
 True
 """
 
+from builtins import object
```

Comment:
We don't need this.

In v11, simply remove the `(object)` in class declarations, that will not hinder future merges from 10 to 11 because it's very localized.

Reply:
OK. I understand that it will be needed in v10. Otherwise it fails 

2017-11-17 16:02:01,769 7451 ERROR 10-mis_reports odoo.addons.mis_builder.tests.test_accounting_none: ` File "/home/jordi/odoo-addons/10.0/OCA/mis-builder/mis_builder/models/accounting_none.py", line 66, in odoo.addons.mis_builder.models.accounting_none
2017-11-17 16:02:01,769 7451 ERROR 10-mis_reports odoo.addons.mis_builder.tests.test_accounting_none: ` Failed example:
2017-11-17 16:02:01,769 7451 ERROR 10-mis_reports odoo.addons.mis_builder.tests.test_accounting_none: `     bool(AccountingNone)
2017-11-17 16:02:01,769 7451 ERROR 10-mis_reports odoo.addons.mis_builder.tests.test_accounting_none: ` Expected:
2017-11-17 16:02:01,769 7451 ERROR 10-mis_reports odoo.addons.mis_builder.tests.test_accounting_none: `     False
2017-11-17 16:02:01,769 7451 ERROR 10-mis_reports odoo.addons.mis_builder.tests.test_accounting_none: ` Got:
2017-11-17 16:02:01,769 7451 ERROR 10-mis_reports odoo.addons.mis_builder.tests.test_accounting_none: `     True